sklearn/tests/test_calibration.py

```diff
@@ -441,27 +456,30 @@ def test_calibration_prob_sum(ensemble):
 def test_calibration_less_classes(ensemble):
     # Test to check calibration works fine when train set in a test-train
```

Comment:
This is a use case where I'm really wondering if this is valid :). But this is here so let's go with it.